<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>manualEntryWindow</class>
 <widget class="QWidget" name="manualEntryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>563</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Manual Entry</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>491</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="midLineWidth">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>12</y>
     <width>503</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../logo/e-triage_manual_entry_form.png</pixmap>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>70</y>
     <width>491</width>
     <height>435</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="nameLabel">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Name:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="tcLabel">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;T.C. No:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="diastBPLabel">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Diastolic Blood Pressure:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="spo2Label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;sp0&lt;/span&gt;&lt;span style=&quot; font-size:10pt; vertical-align:sub;&quot;&gt;2&lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="sysBPLabel">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Systolic Blood Pressure:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="ageLabel">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Age:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="genderLabel">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Gender:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="heartRateLabel">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Heart Rate:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="genderComboBox">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox{
    background-color:  rgb(255,255,255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(0, 0, 120);
    font: bold 14px;
	color: grey;
    min-width: 10em;
    padding: 6px;
}
QComboBox::drop-down 
{
    border: 0px; /* This seems to replace the whole arrow of the combo box */
}
QComboBox::down-arrow {
    image: url(../logo/down_arrow.png);
    width: 14px;
    height: 14px;
}</string>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>Select</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MALE</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../logo/male_gender.png</normaloff>../logo/male_gender.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>FEMALE</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../logo/female_gender.png</normaloff>../logo/female_gender.png</iconset>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="surnameLabel">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Surname:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="temperatureLabel">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Temperature:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nameEdit">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color:  rgb(255,255,255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(0, 0, 120);
    font: bold 14px;
	color: black;
    min-width: 5em;
    padding: 6px;
}</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="surnameEdit">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color:  rgb(255,255,255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(0, 0, 120);
    font: bold 14px;
	color: black;
    min-width: 5em;
    padding: 6px;
}</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="tcEdit">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color:  rgb(255,255,255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(0, 0, 120);
    font: bold 14px;
	color: black;
    min-width: 5em;
    padding: 6px;
}</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="ageEdit">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color:  rgb(255,255,255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(0, 0, 120);
    font: bold 14px;
	color: black;
    min-width: 5em;
    padding: 6px;
}</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="heartRateEdit">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color:  rgb(255,255,255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(0, 0, 120);
    font: bold 14px;
	color: black;
    min-width: 5em;
    padding: 6px;
}</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="spO2Edit">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color:  rgb(255,255,255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(0, 0, 120);
    font: bold 14px;
	color: black;
    min-width: 5em;
    padding: 6px;
}</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="temperatureEdit">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color:  rgb(255,255,255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(0, 0, 120);
    font: bold 14px;
	color: black;
    min-width: 5em;
    padding: 6px;
}</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="diastolicBpEdit">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color:  rgb(255,255,255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(0, 0, 120);
    font: bold 14px;
	color: black;
    min-width: 5em;
    padding: 6px;
}</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="systolicBpEdit">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color:  rgb(255,255,255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(0, 0, 120);
    font: bold 14px;
	color: black;
    min-width: 5em;
    padding: 6px;
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>512</y>
     <width>491</width>
     <height>39</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="1">
       <widget class="QPushButton" name="manualEntryEnterButton">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color:  rgb(255,255,255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(0, 0, 120);
    font: bold 14px;
	color: black;
    min-width: 5em;
    padding: 6px;
}
QPushButton:pressed {
    background-color: rgb(0, 0, 150);
    border-style: inset;
}</string>
        </property>
        <property name="text">
         <string>Enter</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="manualEntryDiscardButton">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color:  rgb(255,255,255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(0, 0, 120);
    font: bold 14px;
	color: black;
    min-width: 5em;
    padding: 6px;
}
QPushButton:pressed {
    background-color: rgb(0, 0, 150);
    border-style: inset;
}</string>
        </property>
        <property name="text">
         <string>Discard</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>13</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
